test-apps/display-test-app/src/frontend/CuratedGoogleMapsSessionClient.ts

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
@aruniverse Obviously this file should not committed but that's shows what the curated content version will look like.